pygsti/modelmembers/operations/fullcptpop.py

```diff
@@ -93,7 +93,7 @@ def _set_params_from_choi_mx(self, choi_mx, truncate):
             Lmx = _np.linalg.cholesky(choi_mx)
 
         #check TP condition: that diagonal els of Lmx squared add to 1.0
-        Lmx_norm = _np.trace(_np.dot(Lmx.T.conjugate(), Lmx))  # sum of magnitude^2 of all els
+        Lmx_norm = _np.linalg.norm(Lmx)  # = sqrt(tr(Lmx' Lmx))
```

Comment:
The square root is just because of using the norm as opposed to the squared norm. It's fine to use either in this context since we're checking that a quantity is near one. In general, if I have an expected value for a norm or squared norm, then I prefer to state it in terms of the norm.